DEVELOPER.md

```diff
@@ -1,39 +0,0 @@
-# WinSW Developer Information
```

Comment:
Could we revert it for now and add a disclaimer that the process is subject to change? Then we can merge it and add patches on the top of it

Reply:
I don't understand why we put this file in the repo. It's internal for maintainers. External contributors should look into *CONTRIBUTING.md*.

Reply:
I am fine to rename it to MAINTAINERS.md and move to docs. Will do as a separate PR